docs/src/components/ComponentDoc/ComponentProp/ComponentPropExtra.tsx

```diff
@@ -1,5 +1,12 @@
 import PropTypes from 'prop-types'
 import * as React from 'react'
+import { Extendable } from 'types/utils'
+
+export interface ComponentPropExtraProps {
+  children?: JSX.Element[]
+  title?: React.ReactNode
+  inline?: boolean
+}
```

Comment:
Now we can safely remove `propTypes` in this file.

Reply:
done, thanks